### ISSUE TITLE
Install SQID

### ISSUE THREAD

**Issue by kyao** [usc-isi-i2/datamart-upload#25]
Install SQID browsing tool for our installation of Wikidata.

https://tools.wmflabs.org/sqid/#/

**Comment by saggu** [comment]
I was able to clone the repository locally, change the sparql endpoint and see the updated endpoint being used.

There are a couple of things going on in SQID,
1. Searches the wikidata api to fetch results for a string.
2. Once user clicks on a search result, uses wikidata api to fetch Qnode  in json format.
3. Uses the json to filter out properties, statements and uses the sparql endpoint to run the queries.
4. Also there are a bunch of json files which is uses internally to fetch some stats about the wikidata as a whole. We can try to create this json files locally(which I havent tried yet). By default, it calls the remote end point (Request URL: https://tools-static.wmflabs.org/sqid/data) to fetch the stats.

So in summary, we should have a local `wikidata api` and a local sparql endpoint to make it work with our local installation of wikibase.

@rongpenl when you installed the wikibase on a server, does it come with an api endpoint as well. The official wikidata ebdpoint is - Request URL: https://www.wikidata.org/w/api.php

FYI @szeke @kyao @dgarijo

**Comment by saggu** [comment]
I am looking into whether we can make it work without having a local `wikidata api` endpoint

**Comment by saggu** [comment]
The file to change the endpoints is

https://github.com/Wikidata/SQID/blob/master/src/api/endpoints.ts

@rongpenl

**Comment by dgarijo** [comment]
Is it possible to make it work without the search? That is adding something like ?Q={QNODE}, where QNODE is the node to retireve in our target endpoint?
